source/features/index.tsx

```diff
@@ -104,6 +106,11 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 	// Options defaults
 	const options = await optionsStorage.getAll();
 
+	// Fetch hotfixes asynchronously
+	const hotfix = await cache.get('hotfix');
```

Comment:
Missing:
https://github.com/sindresorhus/refined-github/issues/3529#issuecomment-687417913

Also it would need a comment, e.g. `// Ignores the cache when loaded locally`

Reply:
I'm sorry I missed this. :confused: :pray:

Reply:
( bringing over the comment from the issue )

> ```
> const hotfix = chrome.manifest.version === '0.0.0' || await cache.get('hotfix');
> ```
> 
> Edit: the code should be exactly this. Firefox reuses the same options when adding the development version, so we cannot rely on or affect the cache.

are you sure it should be `chrome.manifest.version` and not `browser.runtime.getManifest().version`?

I'm getting a TypeScript error that reads `TS2339: Property 'manifest' does not exist on type 'typeof chrome'.`

or did you mean 
```
const hotfix = browser.runtime.getManifest().version === '0.0.0' || await cache.get('hotfix');
```

Reply:
@fregante, ping :point_up: some pointers needed

the reason why I was confused was because you mentioned that the code should be **exactly** that. And also, _Firefox uses the same options_ indicates that this should a chrome-specific value hence it should be `chrome.*`

Reply:
Yes I meant that. That code was just a pointer, like the initial code.